### PR TITLE
Update environment variables examples

### PR DESCRIPTION
**Pull request by chmanie**
## Description

This PR updates the `.env.example` file to account for all environment variables that are needed to run the Dapp locally.

**Changes** 🏗

* Add `PINNING_ROOM` env variable
* Add some more comments

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
-SKIP_GAS_STATION_CONFIRM=true
+# verbose logger
 VERBOSE=true
 
-# API keys for development
+# ethplorer key
 ETHPLORER_API_KEY=freekey
 
-# Infura ID
-INFURA_ID=blahblah
```
Comment (sprusr) [inline .env.example]: Where did this go? It's not required, but useful. Maybe should be in here but commented out by default?

File: .env.example [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
+# ENS suffix
+COLONY_NETWORK_ENS_NAME=joincolony.eth
```
Comment (sprusr) [inline .env.example]: Could benefit from similar explanation to `CHAIN_ID`. `.eth` is used for local and mainnet, `.test` used for goerli.

File: .env.example (hunk 1 above)
Comment (chmanie) [reply .env.example]: OK!

File: .env.example (hunk 2 above)
Comment (chmanie) [reply .env.example]: Will add

**Comment by chmanie** [comment]
@sprusr better?

File: .env.example [hunk 3]
```diff
@@ -1,18 +1,31 @@
...
+# verbose logger
 VERBOSE=true
 
-# API keys for development
+# ethplorer key (use `freekey` for rate limited dev trial)
 ETHPLORER_API_KEY=freekey
 
-# Infura ID
-INFURA_ID=blahblah
+# Uncomment to be rate limited with INFURA
```
Comment (sprusr) [inline .env.example]: Haha this comment! Well either way they don't enforce the rate limit. But just to clarify, if you don't set this in `.env` it uses our default key from colonyJS

Comment (chmanie) [reply .env.example]: We have a default key in colonyJS?

Comment (sprusr) [reply .env.example]: Sure do! https://github.com/JoinColony/colonyJS/blob/develop/packages/colony-js-client/src/lib/InfuraProvider/index.js#L8

It's our "public" key, but if Infura start enforcing the daily request limits, then we're probably going to want a private one for the dApp.

**Pull request merged**